### ISSUE TITLE
[43.6] NPE During AI Movement

### ISSUE THREAD

**Issue by Flibjib** [MegaMek/megamek#844]
Running from MekHQ 43.6

NPE occurs during battle, during AI movement.

Here's the snippet from the mekhq log:

```
Loading wreckMarker image for Harasser Missile Platform (Standard) on the fly.
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException
	at javax.swing.ToolTipManager$insideTimerAction.actionPerformed(ToolTipManager.java:664)
	at javax.swing.Timer.fireActionPerformed(Timer.java:313)
	at javax.swing.Timer$DoPostEvent.run(Timer.java:245)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

```
java.lang.NullPointerException
	at megamek.common.Game.getEntitiesVector(Game.java:1589)
	at megamek.common.Game.getEntitiesVector(Game.java:1566)
	at megamek.common.Compute.isEnemyIn(Compute.java:424)
	at megamek.common.MovePath.compile(MovePath.java:515)
	at megamek.server.Server.processMovement(Server.java:7084)
	at megamek.server.Server.skipCurrentTurn(Server.java:3507)
	at megamek.server.commands.SkipCommand.run(SkipCommand.java:53)
	at megamek.server.Server.processCommand(Server.java:31700)
	at megamek.server.Server.handle(Server.java:31782)
	at megamek.server.Server$1.packetReceived(Server.java:513)
	at megamek.common.net.AbstractConnection.processConnectionEvent(AbstractConnection.java:604)
	at megamek.common.net.AbstractConnection.processPacket(AbstractConnection.java:509)
	at megamek.common.net.AbstractConnection.update(AbstractConnection.java:454)
	at megamek.server.ConnectionHandler.run(ConnectionHandler.java:49)
	at java.lang.Thread.run(Thread.java:745)
```

s: error reading command: null
    Last 50 commands that were sent (oldest first): 80 140 250 80 80 140 250 140 250 80 80 140 250 80 80 140 250 140 250 140 250 80 80 140 250 80 80 140 250 140 250 80 140 250 140 250 80 80 140 250 80 80 140 250 140 250 140 250 80 80 

    Last 50 commands that were received (oldest first): 20 25 70 80 80 80 80 110 80 110 80 80 110 110 80 80 110 80 110 110 80 80 110 110 80 

Contents of Send Queue: 

s:Contents of Send Queue: 
c:Contents of Send Queue: 
c:Contents of Send Queue: 0
s:Contents of Send Queue: 
c:Contents of Send Queue: 
s:Contents of Send Queue: 
client: died
c:Contents of Send Queue: 
client: died
c:Contents of Send Queue: 0
client: died
c:Contents of Send Queue: 
client: died

Here's the bugged save
[bugged.sav.gz.zip](https://github.com/MegaMek/megamek/files/1601975/bugged.sav.gz.zip)
Here it is at round start:
[autosave.sav.gz.zip](https://github.com/MegaMek/megamek/files/1601982/autosave.sav.gz.zip)

Player name: Wolven Winter

**Comment by HammerGS** [comment]
We need the actual log file.  When opening a issue, please review the text in the issue box.

**Comment by Flibjib** [comment]
[mekhqlog.txt](https://github.com/MegaMek/megamek/files/1602052/mekhqlog.txt)

**Comment by NickAragua** [comment]
Fixed with PR #845
